org.freecadweb.FreeCAD.yaml

```diff
@@ -1,11 +1,12 @@
 app-id: org.freecadweb.FreeCAD
-default-branch: stable
```

Comment:
Why not `default-branch: beta`?

Reply:
in general `default-branch` is not needed. It's actually quite annoying when it is set and you build locally.

Reply:
nevermind let's set default-branch and leave branch alone.

Reply:
done